front-packages/akeneo-design-system/src/components/Tiles/Tiles.tsx

```diff
@@ -58,12 +59,22 @@ const TileContainer = styled.div<
           border: 1px solid ${getColor('grey', 80)};
         `}
   &:hover {
```

Comment:
In my opinion more easier to understand that all css hover style is applied only when is not disabled instead of saying which css style is applied on hover when disabled (we agree it totally do the same things)
 
```suggestion
    ${({disabled}) =>
      !disabled &&
      css`
        &:hover {
          border: 2px solid ${getColor('blue', 100)};
          color: ${getColor('blue', 100)};
          margin: 0;
          background: ${getColor('blue', 10)};
        }
      `}
  }
```